Mathlib/Analysis/CStarAlgebra/ContinuousFunctionalCalculus/Range.lean

```diff
@@ -0,0 +1,296 @@
+/-
+Copyright (c) 2025 Jireh Loreaux. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Jireh Loreaux
+-/
+import Mathlib.Analysis.CStarAlgebra.ContinuousFunctionalCalculus.Instances
+import Mathlib.Topology.ContinuousMap.ContinuousSqrt
+
+/-! # range of the continuous functional calculus
+
+This file contains results about the range of the continuous functional calculus, and
+consequences thereof.
+
+# Main results
+
+* `range_cfcHom` and `range_cfcₙHom`: for `RCLike` scalar rings, the range of the continuous
+  functional calculus homomorphism is the elemental subalgebra generated by the given element.
+* `range_cfc_nnreal` and `range_cfcₙ_nnreal`: over the scalar semiring `ℝ≥0`, the range of the
+  continuous functional calculus consists of the nonnegative elements in the elemental `ℝ`-algebra
+  generated by the given element.
+* `Commute.cfc` and `Commute.cfcₙ`: an element commutes with `cfc f a` or `cfcₙ f a` if it
+  commutes with both `a` and `star a`. Specialized versions for `ℝ` and `ℝ≥0` or for
+  `IsSelfAdjoint a` which do not require the user to show the element commutes with `star a` are
+  provided for convenience.
+
+-/
+
+open Topology
+
+open scoped CStarAlgebra
+
+section CFCRangeCommute
+
+theorem range_cfc (R : Type*) {A : Type*} {p : A → Prop} [CommSemiring R] [StarRing R]
```

Comment:
I would expect `range_cfc` to state that `Set.range (cfc (R := R) · a) = elemental 𝕜 a`; this feels like an intermediate result. Could rename this (maybe `range_cfc_eq_range_cfcHom`?) and add this new `range_cfc`? (And likewise for the non-unital case.)